src/atoms/Avatar.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Avatar() {
+    return (
+        <div className="box">
```

Comment:
Maybe you don't need to have an extra `div` around. Try removing it and changing the CSS.

Reply:
I need this box becose it makes `overflow: hidden;`

Reply:
Got it!